JapaneseFilter/sections/specific.txt

```diff
@@ -19,7 +19,7 @@ matome.eternalcollegest.com,ushigyu.net,nilcollection.net,gamers-info.net,appl-y
 ! `###fix_sidebar`
 sugi9532.com,frozen.analyst.jp,5app-ch.com,genshinnews.apexlegends-leaksnews.com###fix_sidebar
 ! `##.code-block`
-bazurecipe.com,frozen.analyst.jp,entano.jp,mainoriti.com,genshinnews.apexlegends-leaksnews.com,crossbike.biz,comicspoil007.com,yoso-walk.net,poire122.com,automaton-media.com,pontako.com##.code-block
+rere.jp,bazurecipe.com,frozen.analyst.jp,entano.jp,mainoriti.com,genshinnews.apexlegends-leaksnews.com,crossbike.biz,comicspoil007.com,yoso-walk.net,poire122.com,automaton-media.com,pontako.com##.code-block
```

Comment:
This blocks thier popup bar on mobile (not sure if is annoyances @zloyden ). I'd go with
```
rere.jp##.rectangle_bnr
rere.jp##.singleTopBnr
```
Note there's already `kashi-mo.com##.rectangle_bnr`

Reply:
> This blocks thier popup bar on mobile

@Yuki2718 unfortunately, I don't have it. Can you show what it block?

>
```
rere.jp##.rectangle_bnr
rere.jp##.singleTopBnr
```

I agree with these rules if they are needed.

Reply:
This one but is already blocked by `##div[data-insertion]` in Base.

<details>
<summary>Screenshot</summary>

![rere](https://user-images.githubusercontent.com/58900598/155506012-87601b9f-53db-433c-ae3a-04a037b7426c.png)

</details>
It links to
```
https://www.rere.jp/beginners/106752/
https://www.rere.jp/beginners/106749/
https://www.rere.jp/beginners/106921/
```
but having looked carefully, they seem to be native-ad articles. So I backtrack my words, `##.code-block` will be fine.

Reply:
Ok, I got it. Thanks for the explanation :)